firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/FirebaseAppDistribution.java

```diff
@@ -135,6 +136,13 @@ public UpdateTask updateIfNewReleaseAvailable() {
 
       cachedUpdateIfNewReleaseTask = new UpdateTaskImpl();
     }
+    if (notOnMainThread()) {
```

Comment:
This sets the exception for the task but it doesn't throw the message into logcat. the log statement before it ensures that logcat shows some error message. Did not want to throw in this method because then the user would have to deal with that situation.  

Other thought could be doing this in the TesterSignInManager since the error occurs on the dialog builder.